dotCMS/src/main/java/com/dotmarketing/db/HibernateUtil.java

```diff
@@ -732,6 +734,14 @@ public static void setTransactionListenersStatus(TransactionListenerStatus statu
 		listenersStatus.set(status);
 	}
 
+	public static ThreadLocal<Boolean> getAsyncCommitListenersFinalization() {
+		return asyncCommitListenersFinalization;

```

Comment:
won't be better just to return the asyncCommitListenersFinalization.get() in order to return just the boolean

Reply:
agree, will change in master PR

Reply:
@jdotcms commited to PR for master